pkg/jobs/trigger_event.go

```diff
@@ -2,21 +2,30 @@ package jobs
 
 import (
 	"github.com/cozy/cozy-stack/pkg/consts"
+	"github.com/cozy/cozy-stack/pkg/permissions"
 	"github.com/cozy/cozy-stack/pkg/realtime"
+	"github.com/labstack/gommon/log"
```

Comment:
We are using `github.com/Sirupsen/logrus` for logs, not `github.com/labstack/gommon/log`.

I've added a check here: https://github.com/cozy/cozy-stack/blob/master/scripts/integration.sh#L18 but it missed it...